libnavigation-base/src/main/java/com/mapbox/navigation/base/options/RoutingTilesOptions.kt

```diff
@@ -177,10 +179,12 @@ class RoutingTilesOptions private constructor(
 
         /**
          * Override tile store instance. It manages downloads and storage for requests to
-         * tile-related API endpoints. For offline/predictive-caching use cases this instance
-         * should be the same that is passed to map resource options.
+         * tile-related API endpoints. For offline/predictive-caching use cases this instance should be
+         * the same that is passed to map resource options. When creating the [TileStore] make sure to call
+         * [TileStore.setOption] with [TileStoreOptions.MAPBOX_ACCESS_TOKEN] and your token.
+         * By default (if `null` is provided here), the [TileStore] will be created with a [filePath] and [NavigationOptions.accessToken].
```

Comment:
the same. should we mention `TileDataDomain.NAVIGATION` ?

Reply:
Same as above.